plasmapy/utils/decorators/helpers.py

```diff
@@ -142,3 +142,51 @@ def wrapper(*args, **kwargs):
         wrapper.__signature__ = inspect.signature(f)
 
     return wrapper
+
+
+def add_lite(flite, attrs=None, scope=None):
```

Comment:
I think this might be better split into two functions, eg. `register_lite` and `register_attrs`. Or, maybe `flite` should also be a kwarg? I'm just thinking that currently attaching an attribute without also attaching a lite function would not be supported, but that seems like it should be possible.

Reply:
Hmm, what use case would you see for that?

Reply:
I don't see one now: just following the "one function one job" principle.

Reply:
It is doing "one job"  I added the ability to add other attributes than "lite", because sometimes you need additional attributes to properly use "lite".  For example, for `thermal_speed.lite` to properly work you also need the coefficients.  So, think of `add_lite` as binding all required "lite" functionality (not just the lite function) to get `<func>.lite` to work.

Reply:
Additionally, to get the documentation to come out correctly and cleanly then the lite function and required attributes needs to be added at the same time.

Reply:
I see that. I guess I'm fine with this implementation now: if we need to add attributes separately in the future I guess we can always break this function apart then.